### ISSUE TITLE
Installation error - "Could not find a part of the path C:\Users\`user`\AppData\Local\brave\packages.betaId'."

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#2232]
From support: tickets/4366

```
2016-06-01 23:56:05> Program: Starting Squirrel Updater: --install . --rerunningWithoutUAC 
2016-06-01 23:56:05> Program: Starting install, writing to C:\Users\ahand\AppData\Local\SquirrelTemp 
2016-06-01 23:56:05> Program: About to install to: C:\Users\ahand\AppData\Local\brave 
2016-06-01 23:56:05> CheckForUpdateImpl: Couldn't write out staging user ID, this user probably shouldn't get beta anything: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\ahand\AppData\Local\brave\packages\.betaId'. 
at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath) 
at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost) 
at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost) 
at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost) 
at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost) 
at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost) 
at System.IO.File.WriteAllText(String path, String contents, Encoding encoding) 
at Squirrel.UpdateManager.CheckForUpdateImpl.getOrCreateStagedUserId() 
2016-06-01 23:56:05> CheckForUpdateImpl: Failed to load local releases, starting from scratch: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\ahand\AppData\Local\brave\packages\RELEASES'. 
at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath) 
at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost) 
at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share) 
at Squirrel.Utility.LoadLocalReleases(String localReleaseFile) 
at Squirrel.UpdateManager.CheckForUpdateImpl.<CheckForUpdate>d__3f.MoveNext() 
2016-06-01 23:56:05> CheckForUpdateImpl: Reading RELEASES file from C:\Users\ahand\AppData\Local\SquirrelTemp 
2016-06-01 23:56:05> CheckForUpdateImpl: First run or local directory is corrupt, starting from scratch 
2016-06-01 23:56:05> ApplyReleasesImpl: Writing files to app directory: C:\Users\ahand\AppData\Local\brave\app-0.10.0 
2016-06-01 23:56:11> IEnableLogger: Failed to install package to app dir: System.AggregateException: One or more errors occurred. ---> System.IO.IOException: Access to the path 'resources' is denied. 
at System.IO.DirectoryInfo.MoveTo(String destDirName) 
at Squirrel.UpdateManager.ApplyReleasesImpl.<>c__DisplayClassaf.<>c__DisplayClassb3.<>c__DisplayClassb5.<installPackageToAppDir>b__ad(DirectoryInfo subdir) 
at Squirrel.Utility.<>c__DisplayClass18`1.<>c__DisplayClass1a.<ForEachAsync>b__17() 
at System.Threading.Tasks.Task.InnerInvoke() 
at System.Threading.Tasks.Task.Execute() 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at Squirrel.Utility.<>c__DisplayClass1e`1.<>c__DisplayClass20.<<ForEachAsync>b__1d>d__22.MoveNext() 
--- End of inner exception stack trace --- 
at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions) 
at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken) 
at Squirrel.UpdateManager.ApplyReleasesImpl.<>c__DisplayClassaf.<>c__DisplayClassb3.<installPackageToAppDir>b__ac(DirectoryInfo ld) 
at Squirrel.EnumerableExtensions.ForEach[TSource](IEnumerable`1 source, Action`1 onNext) 
at Squirrel.UpdateManager.ApplyReleasesImpl.<>c__DisplayClassaf.<<installPackageToAppDir>b__a9>d__b7.MoveNext() 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at Squirrel.Utility.<LogIfThrows>d__50`1.MoveNext() 
---> (Inner Exception #0) System.IO.IOException: Access to the path 'resources' is denied. 
at System.IO.DirectoryInfo.MoveTo(String destDirName) 
at Squirrel.UpdateManager.ApplyReleasesImpl.<>c__DisplayClassaf.<>c__DisplayClassb3.<>c__DisplayClassb5.<installPackageToAppDir>b__ad(DirectoryInfo subdir) 
at Squirrel.Utility.<>c__DisplayClass18`1.<>c__DisplayClass1a.<ForEachAsync>b__17() 
at System.Threading.Tasks.Task.InnerInvoke() 
at System.Threading.Tasks.Task.Execute() 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at Squirrel.Utility.<>c__DisplayClass1e`1.<>c__DisplayClass20.<<ForEachAsync>b__1d>d__22.MoveNext()<---

2016-06-01 23:56:11> Unhandled exception: System.AggregateException: One or more errors occurred. ---> System.AggregateException: One or more errors occurred. ---> System.IO.IOException: Access to the path 'resources' is denied. 
at System.IO.DirectoryInfo.MoveTo(String destDirName) 
at Squirrel.UpdateManager.ApplyReleasesImpl.<>c__DisplayClassaf.<>c__DisplayClassb3.<>c__DisplayClassb5.<installPackageToAppDir>b__ad(DirectoryInfo subdir) 
at Squirrel.Utility.<>c__DisplayClass18`1.<>c__DisplayClass1a.<ForEachAsync>b__17() 
at System.Threading.Tasks.Task.InnerInvoke() 
at System.Threading.Tasks.Task.Execute() 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at Squirrel.Utility.<>c__DisplayClass1e`1.<>c__DisplayClass20.<<ForEachAsync>b__1d>d__22.MoveNext() 
--- End of inner exception stack trace --- 
at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions) 
at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken) 
at Squirrel.UpdateManager.ApplyReleasesImpl.<>c__DisplayClassaf.<>c__DisplayClassb3.<installPackageToAppDir>b__ac(DirectoryInfo ld) 
at Squirrel.EnumerableExtensions.ForEach[TSource](IEnumerable`1 source, Action`1 onNext) 
at Squirrel.UpdateManager.ApplyReleasesImpl.<>c__DisplayClassaf.<<installPackageToAppDir>b__a9>d__b7.MoveNext() 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at Squirrel.Utility.<LogIfThrows>d__50`1.MoveNext() 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task) 
at Squirrel.UpdateManager.ApplyReleasesImpl.<ApplyReleases>d__7b.MoveNext() 
--- End of stack trace from previous location where exception was thrown ---
```

**Comment by simeonf** [comment]
I'm getting the same error on family members Dell XPS laptop running windows 10 Home. 

After getting the error I tried manually making some *.betaId* files - no luck. I also tried running the installer as administrator and continued to have the same error *Couldn't write out staging user ID, this user probably shouldn't get beta anything: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\tracie\AppData\Local\brave\packages\.betaId'.*

After some fumbling around I deleted the *c:\Users\tracie\AppData\Local\brave* and *c:\Users\tracie\AppData\Local\squirreltemp* directories and successfully ran the installer by running as administrator.
